corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -328,6 +328,7 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             this.hasNoItems = options.collection.length === 0;
             this.redoLast = options.redoLast;
             this.selectedCaseIds = [];
+            this.data = options.collection.models;
```

Comment:
I think you already have access to this, possibly in two ways:
* Based on [this example](https://marionettejs.com/docs/master/marionette.collectionview.html#getfilter), I think any `CollectionView` will automatically have `this.collection` available.
* Since when you access this you're already looping over `this.children`, inside of that loop I think you can use `childView.model` (recognizing that this would mean rearranging the `selectAllAction` function a bit to move basically all the logic into that loop).